testutil/filepresencechecker.go

```diff
@@ -0,0 +1,59 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2015-2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package testutil
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/check.v1"
+)
+
+type filePresenceChecker struct {
```

Comment:
I wonder if this (and the IntLessThan, IntEqual etc) checks should simply become something like:
```
c.Check(osutil.FileExists(foo), IsTrue)
c.Check(osutil.FileExists(bar), IsFalse)
c.Check(i < 7, Istrue)
``` 
It means less code and also I personally find this easier to read.

Reply:
While unrelated to the branch, the reason for using those is that there's a useful failure message, otherwise we could do all checks like this:

```
c.Check(whatever boolean expression, Equals, true)
```

Reply:
Aha, this is done so that people don't have to do: `c.Check(bool-expr, Equals, true, Commentf("reasons!"))`